src/components/FindWalletProductTable/FindWalletProductTable.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { Meta, StoryObj } from "@storybook/react"
+
+import FindWalletProductTable from "@/components/FindWalletProductTable"
+
+import walletsData from "@/data/wallets/wallet-data"
```

Comment:
Since the imported file is quite heavy, it will add to the already heavy bundle we have to upload to Chromatic every time we run these tests.

Having said that, we don't really need to replicate the exact same version as in prod, we could just write two or three mocked rows and we could validate the visual tests with a much lighter story.